### PR TITLE
A simple behavior for cvxpy.Parameter

### PR DESCRIPTION
**Pull request by senhalil**
I tried to write the LP file of a CVXPY model which uses Parameters and the `visit_Parameter` introduced in this PR did the trick.

### CODE REVIEW

File: src/cvxpy_gurobi/translation.py [hunk 1]
```diff
@@ -322,6 +322,9 @@ def visit_AddExpression(self, node: AddExpression) -> Any:
     def visit_Constant(self, const: cp.Constant) -> Any:
         return const.value
 
+    def visit_Parameter(self, param: cp.Parameter) -> Any:
+        return param.value
```
Comment (jonathanberthias) [inline src/cvxpy_gurobi/translation.py:326]: We'll want to raise an error if the parameter's value wasn't set yet. You can do it in this PR, or I can do it in a follow-up, it's up to you.

**Review by jonathanberthias — CHANGES_REQUESTED** [review]
The change looks good, I don't think we need anything else. Could you add a test case in `tests/test_atoms.py` please? A very simple one should be enough, something like:
```python
def test_parameter():
    p = cp.Parameter(value=1)
    translater = Translater(gp.Model())
    translater.visit(p)
```

File: src/cvxpy_gurobi/translation.py (hunk 1 above)
Comment (senhalil) [reply src/cvxpy_gurobi/translation.py]: Thanks for the feedback @jonathanberthias ! Added an exception and a test for both happy/non-happy paths in [this](https://github.com/jonathanberthias/cvxpy-gurobi/pull/67/commits/1b832e6a96b06ca162889ed0dc4773a797092c70) commit.

**Comment by jonathanberthias** [comment]
There's a linting error, but other than that it's perfect! Thanks a lot @senhalil 👍

**Comment by jonathanberthias** [comment]
I've went ahead and pushed some formatting changes to make the linter happy. Thanks a lot for your contribution @senhalil, much appreciated 🤗 

(There's an unrelated error in CI, I'll fix that and then merge this PR)

**Review by jonathanberthias — APPROVED** [review]

**Comment by senhalil** [comment]
@jonathanberthias thanks for fixing the linter, I couldn't get back to it on time. 

(Unrelated topic: what would you say for constraint names? My team feels the need and we ask ourselves if we can somehow add this feature to cvxpy and make use of it via cvxpy-gurobi.)

**Comment by jonathanberthias** [comment]
> what would you say for constraint names? My team feels the need and we ask ourselves if we can somehow add this feature to cvxpy and make use of it via cvxpy-gurobi.

AFAIK, `cvxpy` doesn't really expect you to set names for constraints, but by quickly looking at the source code, I think it should be possible to override the `constr_id` which is what `cvxpy-gurobi` uses.
```python
my_constr = cp.sum(w) == 1
my_constr.constr_id = "soundess"
```

I haven't tested this, but I don't there would be any problem. Except maybe you have to ensure the `constr_id`s are unique across all constraints in the problem?

**Comment by senhalil** [comment]
You are right, my suggestion assumes some dev on cvxpy side.

That feels a bit tacky and I am afraid mixing IDs and names might problematic. 

I was thinking of something like expression.name, similar to variable.name on the cvxpy side and then using it on cvxpy-gurobi side.

**Comment by jonathanberthias** [comment]
I found this old issue on cvxpy which unfortunately never led to anything 🤷  https://github.com/cvxpy/cvxpy/issues/322.

**Pull request merged**